symfony/mercure-bundle/0.3/manifest.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
This can be a symlink to the file in `0.1`.

Reply:
fixed